bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/ZZjj_llvvjj_ewk_aQGC/ZZjj_llvvjj_ewk_aQGC_proc_card.dat

```diff
@@ -0,0 +1,16 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+
+import model SM_Ltotal_Ind5v2020v2_UFO 
+
+define p = g u c d s u~ c~ d~ s~
+define j = p
```

Comment:
I see in your model file it's set to 5f scheme as default.
Is there a reason to change this to 4f?

Reply:
@sihyunjeon 
We found b did not contribute significantly to our signal region, and removing it could speed up the generation process. Is it recommended to use 5f instead of 4f? If so, we'll make that change quickly. Thanks!

Reply:
@sihyunjeon we have switched to 5f in the new commit.